tests/test_jointcal.py

```diff
@@ -269,6 +271,76 @@ def test_fit_astrometry_writeChi2(self):
+        associations = lsst.jointcal.Associations()
+        associations.addCcdImage(struct.ccdImageList[0])
+        associations.addCcdImage(struct.ccdImageList[1])
+        associations.computeCommonTangentPoint()
+        bbox = struct.bbox
+
+        circle = associations.computeBoundingCircle()
+        center = lsst.geom.SpherePoint(circle.getCenter())
+        radius = lsst.geom.Angle(circle.getOpeningAngle().asRadians(), lsst.geom.radians)
+        points = [lsst.geom.SpherePoint(struct.skyWcs[0].pixelToSky(lsst.geom.Point2D(x)))
+                  for x in bbox.getCorners()]
+        points.extend([lsst.geom.SpherePoint(struct.skyWcs[1].pixelToSky(lsst.geom.Point2D(x)))
+                      for x in bbox.getCorners()])
+        self._checkPointsInCircle(points, center, radius)
+
+    def testPointsRA0(self):
+        """Test for CcdImages crossing RA=0: this is DM-19802"""
+        # Use the same pixel origins as the cfht_minimal data, but put the sky origin at RA=0
+        crpix = lsst.geom.Point2D(931.517869, 2438.572109)
+        cd = np.array([[5.19513851e-05, -2.81124812e-07],
+                      [-3.25186974e-07, -5.19112119e-05]])
+        crval1 = lsst.geom.SpherePoint(0.01, -0.01, lsst.geom.degrees)
+        crval2 = lsst.geom.SpherePoint(-0.01, 0.01, lsst.geom.degrees)
+        wcs1 = lsst.afw.geom.makeSkyWcs(crpix, crval1, cd)
+        wcs2 = lsst.afw.geom.makeSkyWcs(crpix, crval2, cd)
+
+        # Put the visit boresights at the WCS origin, for consistency
+        visitInfo1 = lsst.afw.image.VisitInfo(exposureId=30577512,
+                                              date=DateTime(65321.1),
+                                              boresightRaDec=wcs1.getSkyOrigin())
+        visitInfo2 = lsst.afw.image.VisitInfo(exposureId=30621144,
+                                              date=DateTime(65322.1),
+                                              boresightRaDec=wcs1.getSkyOrigin())
+
+        struct = lsst.jointcal.testUtils.createTwoFakeCcdImages(fakeWcses=[wcs1, wcs2],
+                                                                fakeVisitInfos=[visitInfo1, visitInfo2])
+        lsst.log.setLevel('jointcal', lsst.log.DEBUG)
+        associations = lsst.jointcal.Associations()
+        associations.addCcdImage(struct.ccdImageList[0])
+        associations.addCcdImage(struct.ccdImageList[1])
+        associations.computeCommonTangentPoint()
+        bbox = struct.bbox
+
+        circle = associations.computeBoundingCircle()
+        center = lsst.geom.SpherePoint(circle.getCenter())
+        radius = lsst.geom.Angle(circle.getOpeningAngle().asRadians(), lsst.geom.radians)
+        points = [lsst.geom.SpherePoint(wcs1.pixelToSky(lsst.geom.Point2D(x)))
+                  for x in bbox.getCorners()]
+        points.extend([lsst.geom.SpherePoint(wcs2.pixelToSky(lsst.geom.Point2D(x)))
+                      for x in bbox.getCorners()])
+        self._checkPointsInCircle(points, center, radius)
```

Comment:
I can't help but notice that lines 327-341 are identical to 290-304, as best as I can tell.  Is there a way to consolidate this?